src/EnergyPlus/MixedAir.cc

```diff
@@ -4146,6 +4267,10 @@ namespace MixedAir {
 			OAController( OAControllerNum ).OAMassFlow = min( OAController( OAControllerNum ).OAMassFlow, OAController( OAControllerNum ).MaxOAMassFlowRate );
 		}
 
+		// Update demand managers here and before EMS override becuase we want EMS to have higher priority than demand manager
+		if ((OAController(OAControllerNum).ManageDemand) && (OAController(OAControllerNum).OAMassFlow > OAController(OAControllerNum).DemandLimitFlowRate))
```

Comment:
The DR is applied only for Simulation period, not for Warmup and Sizing. Global flags can be used to achieve that.